core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -3537,110 +3537,90 @@ class ViewMapping<
 namespace Kokkos {
 namespace Impl {
 
-template <unsigned, class MapType>
-KOKKOS_INLINE_FUNCTION bool view_verify_operator_bounds(const MapType&) {
-  return true;
+template <class Map, class... Indices, std::size_t... Enumerate>
+KOKKOS_FUNCTION bool check_bounds(Map const& map,
```

Comment:
```suggestion
KOKKOS_FUNCTION bool bounds_satisfied(Map const& map,
```
to make it clearer that we are returning a `bool`.

Reply:
I renamed it to `within_range`.  Let me know if you still do not like the name.